src/Wrapper.tsx

```diff
@@ -194,14 +194,16 @@ export default function Wrapper({ title, page, children }: WrapperProps): ReactE
                 </a>
               </Link>
             </Box>
-            <Typography
-              variant='h6'
-              noWrap
-              component='div'
-              sx={{ flexGrow: 1, ml: 2, display: { xs: 'none', md: 'flex' } }}
-            >
-              Bailo
-            </Typography>
+            <Link href='/' passHref>
+              <Typography
+                variant='h6'
+                noWrap
+                component='div'
+                sx={{ flexGrow: 1, ml: 2, display: { xs: 'none', md: 'flex', cursor: 'pointer' } }}
```

Comment:
We shouldn't be defining `cursor` on our typography, it should be caused by our `Link` element.  Do we need to add it here?

Reply:
Yes - because I'm using the Typography component underneath. Perhaps I can remove it and just use the text without Typography?

Reply:
<img width="504" alt="Screenshot 2022-10-21 at 15 03 33" src="https://user-images.githubusercontent.com/89583054/197214677-49e86668-20f4-4993-9f4c-507485f93a1b.png">

It seems to be using a div instead of a link.  We'll need to fix this to conform with accessibility requirements.

Reply:
I think this might because of the typography thing. I think the best thing to do is use an anchor tag and change the styling of it?